src/tests/missions.test.js

```diff
@@ -0,0 +1,80 @@
+import missionsReducer, {
+  loadMissions,
+  joinMission,
+  leaveMission,
+} from '../redux/Missions/missions';
+
+describe('Unit tests for missions', () => {
+  jest.mock('../redux/missions/missions');
```

Comment:
[OPTIONAL] This keeps failing, I'm going to change the requirement to optional because it's just a typo error and after that, it works normally, but I highly encourage you guys to fix it before merging to prod as it's a great practice no commiting code that is not ready for production or with errors.

![image](https://user-images.githubusercontent.com/79114233/146296801-4a5afc43-9c31-4a35-9d94-a07b91dd9c1b.png)

Reply:
Thanks a lot @Daniel I am going to fix it right now! I am a little confused about how is that is passing in our respective ends... it's weird when you don't know why your code doesn't work but is even weirder when you don't know why is working haha